crates/bevy_asset/src/loader.rs

```diff
@@ -256,7 +256,7 @@ impl ErasedLoadedAsset {
 
     /// Cast this loaded asset as the given type. If the type does not match,
     /// the original type-erased asset is returned.
-    #[allow(clippy::result_large_err)]
+    #[expect(clippy::result_large_err, reason = "Function returns `Self` on error.")]
     pub fn downcast<A: Asset>(mut self) -> Result<LoadedAsset<A>, ErasedLoadedAsset> {
```

Comment:
Do we not prefer to use `Self` in impl blocks?

Reply:
You could also mention that the "large error variant" is only 10% bigger than the okay variant.

Reply:
Generally we don't have strong feelings. I prefer Self here though.